partner_multi_relation_parent/models/res_partner.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import api, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    # called without parameters from the init hook
+    def find_current_relation(self, left, type_relation, right):
+        par_rel_mod = self.env['res.partner.relation']
+        return par_rel_mod.search([
```

Comment:
there's a high chance that this is already in the cache, so better use `self.relation_all_ids` and restrict to what you search for. I also don't see the point of this function given you use it only once and there's no obvious reason why other modules should want to override exactly this function